appium/appiumdriver-framework-core/pom.xml

```diff
@@ -71,9 +64,12 @@
             <version>${browserstack.local.version}</version>
         </dependency>
     </dependencies>
-
     <build>
         <plugins>
+            <plugin>
```

Comment:
We can exclude the gpg-plugin from the build section. Since it's a part of the profile